recipe/run_test.bat

```diff
@@ -4,7 +4,7 @@
 :: It's weird that ninja is needed because it's already in run and test/requires.
 :: opencv-python-headless is needed only for tests, in run is used py-opencv.
 ::- pip install ninja "opencv-python-headless<=4.5.4.60"
-::%PYTHON% -m pip check || (exit 1)
+pip check easyocr || exit 0
```

Comment:
so if pip check easyocr fails ( return non-zero ) we exits this script with zero code which I believe means success

Reply:
I wonder if it should be `pip check easyocr || cmd /K "exit /b 0"` instead of `pip check easyocr || exit 0`

Reply:
Yes, that is the point of the command. It's been placed there solely to print the output to the logs, which can be used to verify that the comments above (which explain why the check fails) are true. `exit 0` is correct here.

Reply:
On a side note, I find it deeply disrespectful that comment implies that I don't know what `exit 0` is, or how piping works.

Reply:
it is not that you don't know what it means, it is just that it is not a good practice. if something fails we need to find why, there' nothing personal or disrespectful in my comments imho.

Reply:
@sumit0190 is not what the review are for find even silly mistakes ?, comment does not mean the person does know what he does, just simply mistake which can happen to anyone.
But in this particular case I agree with @rafaelmartins it is not good practice, most of us if the build is successful will not look deeply in the logs, not to mention that CI would happily get through even though pip check would fail.

Reply:
If you find my comment disrespectful, I apologize. That was never my intention

Reply:
@M-Waszkiewicz-Anaconda I care about the review - that's what I am here for, of course. Most times, even though I might disagree with your assessment, I am always willing to make changes to incorporate everyone's feedback and keep this a respectful process.
But I don't accept unnecessarily patronizing comments either. In this case, one could've simply said "I disagree with this, this is bad practice; would you mind explaining this addition?" or something along those lines. Adding an explanation for how or why that fails - especially for something so trivial, which I _deliberately_ added - seems to imply that I don't know what I am doing.
In any case, thank you and @rafaelmartins for the feedback; I will make this change.

Reply:
Thank you @sumit0190

Reply:
Fixed.

Reply:
Thank you